File: Client/src/app/models/people.ts
export class People {
  private _id: number;
  private _name: string;
  get name(): string {
    return this._name;
  }

  set name(value: string) {
    this._name = value;
  }

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  constructor(data) {
    Object.assign(this, data);
  }
}

File: Server/PoC.Data.Models/People.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace PoC.Data.Models
{
    [Table("People")]
    public class People
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key]
        public int Id { get; set; }

        [MaxLength(75)]
        public string Name { get; set; }

        public virtual ICollection<Favourite> Favourites { get; set; } = new List<Favourite>();
    }
}